### ISSUE TITLE
Facilities - Sprint 56

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#8714]
<details><summary>Background</summary>

🥅   — Sprint goals (pending Sprint planning)
💡 — Design
💧  — Drupal engineering
🎉 — FE
🎓  — Training / Change management / KB
📟 — Helpdesk
🔮  —  Planning / roadmap / epic refinement
📊  —  Analytics
🚰  — Service design
🌱 — Good first task

</details>

## Transition items

- [ ] Lighthouse API
- [ ] Mapbox security issue department-of-veterans-affairs/va.gov-team-sensitive#454
- [ ] Facility Locator
  - [ ] Provider Profile Management System - Data source for Community care providers, Urgent Care, Community pharmacies. VAOS (VA Online Scheduling) also consumues this endpoint in Vets API
  - [ ] Bounding box to Radius got rolled back to bounding box, possibly low priority to go back to radius.
  - [ ] Pull VA health connect phone number from Lighthouse to Facility locator (can be hardcoded in the meantime as new VA health connect numbers are getting added)

## department-of-veterans-affairs/va.gov-team#7622 Lovell

Background: See epic https://github.com/department-of-veterans-affairs/va.gov-team/issues/34022

### Front-end

Some  design background is in https://github.com/department-of-veterans-affairs/va.gov-team/issues/38196
- [x] department-of-veterans-affairs/va.gov-team#8715 
- [x] department-of-veterans-affairs/va.gov-team#8716 

### CMS 
(question - how far should we proceed until Veteran/Service member research has progressed?)
- [ ] department-of-veterans-affairs/va.gov-team#7626 (sprint 55)
- [ ] Create proof of concept
- [ ] CMS design discovery epic
  - [ ] Updated IA artifact to represent syndicated vs unique content and assist with designing CMS editor workflow
- [ ] Refine epic department-of-veterans-affairs/va.gov-team#1939 (for DoD facilities)
  - [ ] department-of-veterans-affairs/va.gov-team#1940 
  - [ ] department-of-veterans-affairs/va.gov-team#1941 
  - [ ] department-of-veterans-affairs/va.gov-team#1942 
  - [ ] department-of-veterans-affairs/va.gov-team#1943  

### department-of-veterans-affairs/va.gov-team#6931 Deprecate table-based hours in favor of office hours field
(_These appear farther down in lighthouse push section as they are entangled there._)
Which issues are ready to go here? Do we need some updates to training and/or change management?
- [x] https://github.com/department-of-veterans-affairs/va.gov-team/issues/39436
- [x] department-of-veterans-affairs/va.gov-team#8708
- [x] department-of-veterans-affairs/va.gov-team#8709
- [x] department-of-veterans-affairs/va.gov-team#8426

TBD if needed
- [ ] Update KB article
- [ ] Change management - a dozen or so instances where hours data was not migrateable.

###  department-of-veterans-affairs/va.gov-team#3151 3 VAMC Top task pages

Front-end (epics)
- [x] https://github.com/department-of-veterans-affairs/va.gov-team/issues/37367

Some Cerner work to do here
- [x] https://github.com/department-of-veterans-affairs/va.gov-team/issues/37365
- [x] https://github.com/department-of-veterans-affairs/va.gov-team/issues/37366

CMS
- [x] department-of-veterans-affairs/va.gov-team#8337 
- [x] department-of-veterans-affairs/va.gov-team#8648 

## Events V2 and Recurring events

Issues in epic https://github.com/department-of-veterans-affairs/va.gov-team/issues/42556

## department-of-veterans-affairs/va.gov-team#7158 Cerner cutover strategy 

[needs issues] Update various CTAs for "Staging to Cerner"

## department-of-veterans-affairs/va.gov-team#7743 Resources and support pilot

This is a PW product but for continuity the pilot may continue by Facilities team for now. 

### department-of-veterans-affairs/va.gov-team#7624 Support lighthouse with unique ID for health service terms / department-of-veterans-affairs/va.gov-team#5017  Push facility services to lighthouse

Blocked by Lighthouse v1 API
- [ ] department-of-veterans-affairs/va.gov-team#8019 done but not merged, blocked by Lighthouse 
- [ ] department-of-veterans-affairs/va.gov-team#8017  blocked by Lighthouse AND
    - [ ] Field Office Hours hardening https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6931
      - [x] script the migration https://github.com/department-of-veterans-affairs/va.gov-cms/pull/8533
      - [x] run the migration on prod to populate data for FE 
      - [ ] FE builds new display  https://github.com/department-of-veterans-affairs/va.gov-team/issues/39436 
      - [ ] After FE goes live(same day) Hide field `field_facility_service_hours` and show 'field_office_hours' in cms  department-of-veterans-affairs/va.gov-team#8550 
      - [ ] Re-run script on prod `drush scr scripts/content/VACMS-6931-switch-service-locations-office-hours.php`
      - [ ] Notify editors / process bad entries from the outliers.csv department-of-veterans-affairs/va.gov-team#8708
      - [x] department-of-veterans-affairs/va.gov-team#8709

Possible to add to Lighthouse V0 API
- [x] Stretch: department-of-veterans-affairs/va.gov-team#8565 
- [x] Stretch: department-of-veterans-affairs/va.gov-team#7726
- [x] Stretch: department-of-veterans-affairs/va.gov-team#7279
- [x] Stretch: department-of-veterans-affairs/va.gov-team#6318
- [ ] Stretch: department-of-veterans-affairs/va.gov-team#4769 

## Operations & Maintenance (Steve, Jay, User support team)

- [ ] department-of-veterans-affairs/va.gov-team#7728 and its many children
